src/app-directory/specification/examples/application/fdc3-tradingview-chart-orig.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+    appId: 'fdc3-tradingview-chart-orig',
+    name: 'FDC3 TradingView Chart (orig)',
+    type: 'browser',
+    details: {
+        url: 'https://appd.kolbito.com/demos/tradingview-chart/index.html'
```

Comment:
I don't think we should include an example that's contingent on two external domains. 

FDC3 Workbench might make a suitable replacement which is under our own domain